### DISCUSSION TITLE
Is table column resizing currently available?

### DISCUSSION THREAD

**Discussion by KrzysztofSixtus** [ant-design/ant-design#36466] [Q&A]
In the component overview, there is no info on column resizing:

https://ant.design/components/table/

However, it can still be found here:

https://ant.design/components/table/#components-table-demo-resizable-column

What is the status of the feature?

Another question about the availability of column resizing hasn't been answered #29492

**Comment by abenhamdine** [comment]
> What is the status of the feature?

there is no such feature built in antd table, but as shown in demo, you can implement it with react-resizable

**Reply by KrzysztofSixtus** [reply]
Yes, I am aware that there isn't a built-in feature, but why is the demo hidden on the Table component page? Is there a specific reason for this (not working well with react-resizable etc.)?
